openmc/mgxs/mgxs.py

```diff
@@ -548,26 +548,14 @@ def get_nuclide_density(self, nuclide):
         float
             The atomic number density (atom/b-cm) for the nuclide of interest
 
-        Raises
```

Comment:
I'm curious if we should also return 0.0 from the `get_xs(...)` method for nuclides which are not found in an MGXS domain? I don't see a use case for this for myself right now, but perhaps the good folks at ANL-NE who are stressing this capability might.